lib/elixir/lib/calendar.ex

```diff
@@ -349,10 +440,18 @@ defmodule Date do
     {year, month, day}
   end
 
+
+  def to_erl(%{calendar: calendar, year: year, month: month, day: day} = date) do
+    result_date = convert!(date, Calendar.ISO)
```

Comment:
Please use pattern matching instead of multiple access like date.year, date.month, etc.